metadata_backend/helpers/validator.py

```diff
@@ -15,7 +14,7 @@ def validate(xml_content, schema_name, schema_loader):
         Validates xml string against schema found with schema_name.
 
         :param xml_content: xml to be validated
-        :param schame_name: schema used for validation
+        :param schema_name: schema used for validation
```

Comment:
to avoid spelling issue this could be part of your future CI integration: https://github.com/client9/misspell